api/UIEvent.json

```diff
@@ -255,13 +255,15 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/UIEvent/layerX",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "1",
+              "version_removed": "45"
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "18",
+              "version_removed": "45"
             },
             "edge": {
-              "version_added": "12"
+              "version_added": false
```

Comment:
Many of the changes from a specific version to false in this PR have me wondering... since `UIEvent` is an abstract base class and browsers don't fire any events that are direct `UIEvent` instances, maybe `layerX` was just on `MouseEvent` or something and should be considered supported? In other words, it might be a case of https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#apis-moved-on-the-prototype-chain

Can you do some more checking of this?

Reply:
I've reverted the changes from true to false for the time being to save some reviewing time.  I'll revisit these changes in a later PR!